### PR TITLE
Feature/aws deployment

### PR DESCRIPTION
**Pull request by flomotlik**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+MAINTAINER maintainers@codeship.com
+
+RUN apk --update add \
+    python \
+    py-pip && \
+  pip install awscli && \
+  apk --purge del py-pip && \
+  rm var/cache/apk/*
+
+ONBUILD COPY . /app
```
Comment (mlocher) [inline Dockerfile]: Do we need to create the `/app` directory first?

**Comment by mlocher** [comment]
:shipit: (as `COPY` creates the directory if it doesn't exist already)

**Comment by mlocher** [comment]
We will later add additional script for ElasticBeanstalk deployment (similar to our Heroku deployment script)?

**Comment by flomotlik** [comment]
Yes exactly. Putting this in place now so we have the basic AWS deployment container and docs ready. Getting all the other docs in place and then working on ElasticBeanstalk and ECS deployment scripts.

**Pull request merged**